### PR TITLE
[PLAYER- GetIsPlayerDrivingOnHighway]: Resolved Name & Documented Native

### PR DESCRIPTION
**Pull request by JayPaulinCodes**
## Summary
Using a joaat resolver and hasher (https://joaat.sh/) I found elsewhere in the project, the name of the native was resolved and then added to a script to test assumed functionality based on the native name. The native was then documented accordingly

## Validating Example(s)
- Video showcasing the functionality of the native: https://youtu.be/n-pSetC2AOQ
- Code used for the validating examples:
```lua
function GET_IS_PLAYER_DRIVING_ON_HIGHWAY(player)
    return N_0x5fc472c501ccadb3(player)
end

local loopOn = false
RegisterCommand("getIsPlyDrivingOnHwy_dev_start", function(soruce, args)
    Citizen.CreateThread(function()
        loopOn = true
        while loopOn do
            Citizen.Wait(500)
            print("getIsPlyDrivingOnHwy_dev_start", GET_IS_PLAYER_DRIVING_ON_HIGHWAY(PlayerId()))
        end
    end)
end)

RegisterCommand("getIsPlyDrivingOnHwy_dev_stop", function(soruce, args)
    loopOn = false
end)
```

## Contribution Guidelines Acknowledgement
Before you submit this PR, please make sure:
- [x] You have read the contribution guidelines
- [x] You include an example that validates your change
- [x] Your English is grammatically correct

### CODE REVIEW

**Comment by coalaura** [comment]
hell yeah! glad to see the tool i built is coming to some good use :D

File: PLAYER/GetIsPlayerDrivingOnHighway.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+---
+## GET_IS_PLAYER_DRIVING_ON_HIGHWAY
+
+```c
+// 0x5FC472C501CCADB3 0x46E7E31D
+BOOL GET_IS_PLAYER_DRIVING_ON_HIGHWAY(Player playerId);
+```
+
+```
+Appears once in "re_dealgonewrong"
```
Comment (ahcenezdh) [inline PLAYER/GetIsPlayerDrivingOnHighway.md:13]: I don't think it's important to put this maybe you should remove it

File: PLAYER/GetIsPlayerDrivingOnHighway.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Return value
+Returns a boolean value representing if the player is driving on a highway.
+
+## Examples
+```lua
+-- Get and store the local player index
+local playerId = PlayerId()
+
+if GetIsPlayerDrivingOnHighway(playerId) then
```
Comment (ahcenezdh) [inline PLAYER/GetIsPlayerDrivingOnHighway.md:27]: You could add JS/C# support too !

Comment (JayPaulinCodes) [reply PLAYER/GetIsPlayerDrivingOnHighway.md:27]: I don't control support for the natives, simply just contributed to the documentation.

**Pull request merged**